lib/extractor/docker-archive/index.ts

```diff
@@ -15,7 +16,13 @@ export function getImageIdFromManifest(
   manifest: DockerArchiveManifest,
 ): string {
   try {
-    return manifest.Config.split(".")[0];
+    const imageId = manifest.Config.split(".")[0];
+    const hashAlgorithmPrefix = `${HashAlgorithm.Sha256}:`;
```

Comment:
what happens if they use a different hashing algorithm?

Reply:
That's a good question. From what I read [here](https://windsock.io/explaining-docker-image-ids/#:~:text=A%20Docker%20image's%20ID%20is,when%20an%20image%20is%20pushed), it looks like the value of manifest.Config is the imageId, which is a SHA256 hash of the image's JSON configuration object.
I didn't find a formal documentation, but added my  findings [here](https://snyk.slack.com/archives/C01EPSMGZML/p1618416730094000?thread_ts=1618396754.091000&cid=C01EPSMGZML).

Reply:
your link looks like a blog post based on Docker, not OCI's image spec.
I'd stick to the spec - if the spec doesn't stick to a specific hashing algorithm, why should we?